### PR TITLE
112 Update to `convert_model` Python API and PTQ API

### PR DESCRIPTION
**Pull request by igor-davidyuk**
This PR takes advantage of the new MO and NNCF API. Additionally, it includes a pip package installation step, making this notebook independent of the global repo requirements file. A Google Colab badge has been added as well.

BEFORE MERGING:

1. Update the Google Colab link to point to the main repository.
2. Consider the method used for importing notebooks_utils.

### CODE REVIEW

File: notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
```
Comment (eaidova) [inline notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    !pip install --upgrade pip &amp;&amp; pip install \
</pre><p>only addiional requirements not specified in requirements.txt should be specified, also </p><p>torch==1.13.1+cpu" "torchvision==0.14.1+cpu" -f https://download.pytorch.org/whl/torch_stable.html does not work for macos</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (eaidova) [inline notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb:1]: <p>here no training pipeline in notebook, I think its mention should be removed</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (eaidova) [inline notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb:1]: <p>I think we need to provide link on NNCf and PTQ API https://docs.openvino.ai/latest/basic_qauntization_flow.html#doxid-basic-qauntization-flow</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (igor-davidyuk) [reply notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb:1]: <p>done!</p>

Comment (igor-davidyuk) [reply notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb:1]: removed the training pipeline mention

**Review by eaidova — APPROVED** [review]
